packages/databases-collections/src/modules/create-namespace.ts

```diff
@@ -402,17 +403,20 @@ export const createNamespace = (
         expires: !!data.options.expireAfterSeconds,
       };
 
-      track(
-        `${kind} Created`,
-        trackEvent,
-        connectionRepository.getConnectionInfoById(connectionId)
-      );
+      const connectionInfo =
+        connectionRepository.getConnectionInfoById(connectionId);
+
+      track(`${kind} Created`, trackEvent, connectionInfo);
 
       globalAppRegistry.emit('collection-created', namespace, {
         connectionId,
       });
       workspaces.openCollectionWorkspace(connectionId, namespace, {
         newTab: true,
+        initialSubtab:
+          connectionInfo && connectionSupports(connectionInfo, 'globalWrites')
```

Comment:
connectionSupports won't be sufficient. we'll need to be filtering out the admin/config/local collections and also views. please sync with @djechlin-mongodb
more context: https://mongodb.slack.com/archives/C03DU5ATTB6/p1729869659313749

Reply:
sorry for only realising this now 🙈

Reply:
You can't create views through this modal and I'm pretty sure that even db users with admin roles can't create collections inside special-ish namespaces (but for anything that is defined as special-ish by mongodb-ns package we probably can add a filter here, yeah)

Reply:
ah right, thanks @gribnoysup !

Reply:
added in [0d124af](https://github.com/mongodb-js/compass/pull/6405/commits/0d124af0cf4fe8d7b81d29e8ef30c2b48bccbdbf)